bridge/bridgeapp/views.py

```diff
@@ -27,9 +28,13 @@ def get(self, request):
 
 class BridgeCategoryView(View):
     """Get Thread table from database and display threads associated with Category selected on Home Page"""
-    def get(self, request, category_id, category_slug):  #category_slug needed for UI
+
+    def get(self, request, category_id, category_slug):  # category_slug needed for UI
```

Comment:
A function or method should have a docstring describing its purpose.

Here it's "processing GET action".